.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             - uses: pnpm/action-setup@v4
 
             - name: Use Node.js 18.x
-              uses: actions/setup-node@v3
+              uses: actions/setup-node@v4
               with:
                   node-version: 18
```

Comment:
Does this fix the warning? Since the node version is still set to 18 here?

Reply:
It's not about the version we use, but the version that `actions/setup-node@v3` used (Node 16): https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

Reply:
Ah okay, now I understand